Dockerfile

```diff
@@ -40,7 +40,7 @@ RUN GOOS=${TARGETOS} GOARCH=${TARGETARCH} go build -o /function .
 
 # Produce the Function image. We use a very lightweight 'distroless' image that
 # does not include any of the build tools used in previous stages.
-FROM gcr.io/distroless/base-debian12 AS image
+FROM gcr.io/distroless/base-debian12:nonroot AS image
```

Comment:
looks like a fair bit of other functions use `static-*` instead. should that change be made here too?

* https://github.com/crossplane-contrib/function-patch-and-transform/blob/main/Dockerfile#L25
* https://github.com/crossplane-contrib/function-go-templating/blob/main/Dockerfile#L41
* https://github.com/crossplane-contrib/function-auto-ready/blob/main/Dockerfile#L25

Reply:
Yes that makes sense, I updated the image. Can you have another look, please?